luxonis_train/nodes/backbones/recsubnet/recsubnet.py

```diff
@@ -0,0 +1,32 @@
+from typing import Tuple
+
+from torch import Tensor
+
+from luxonis_train.nodes.base_node import BaseNode
+
+from .blocks import Decoder, Encoder
+
+
+class RecSubNet(BaseNode[Tensor, Tuple[Tensor, Tensor, Tensor]]):
+    in_channels: int
+    out_channels: int
+    base_width: int
+
+    def __init__(
+        self,
+        in_channels=3,
+        out_channels=3,
+        base_width=128,
+        **kwargs,
+    ):
+        super().__init__(**kwargs)
```

Comment:
Let's add docstrings and let's also include link to paper and original implementation.